src/nvim/memory.c

```diff
@@ -596,9 +596,7 @@ void free_all_mem(void)
 # endif
 
   /* message history */
-  for (;; )
-    if (delete_first_msg() == FAIL)
-      break;
+  while(msg_delete_first()) {;}
```

Comment:
As far as I've seen this is the first instance of the `{;}` pattern, I though we were using just `{}`.